### PR TITLE
DOCS-15979 new-auto-merge-shell-helpers

### PR DESCRIPTION
**Pull request by jason-price-mongodb**
## DESCRIPTION

4 helper links.

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/docs/docsworker-xlarge/DOCS-15979-new-auto-merge-shell-helpers/reference/method/#sharding

## JIRA

https://jira.mongodb.org/browse/DOCS-15979

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64948202423952aeb970d154

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/methods.txt [hunk 1]
```diff
@@ -1254,6 +1254,13 @@ Sharding Methods
 
        .. versionadded:: 4.4
 
+   * - :upcoming:`sh.disableAutoMerger()
```
Comment (jmd-mongo) [inline source/reference/methods.txt:1257]: [question]

What's this directive? I don't believe I've seen this before and don't understand how it differs from `:ref:` in a case like this.

**Review by jmd-mongo — COMMENTED** [review]
Hi @jason-price-mongodb,

Thanks for the updates. I've left a question for you.

Thanks,
Joe

File: source/reference/methods.txt (hunk 1 above)
Comment (jmd-mongo) [reply source/reference/methods.txt:1257]: Resolved via Slack.

**Review by jmd-mongo — APPROVED** [review]
This one LGTM. Thanks, @jason-price-mongodb!

**Review by pierlauro — APPROVED** [review]

**Pull request merged**